compiler/x/codegen/OMRMachine.cpp

```diff
@@ -270,15 +270,15 @@ OMR::X86::Machine::findBestFreeGPRegister(TR::Instruction   *currentInstruction,
          first = TR::RealRegister::FirstGPR;
          last  = TR::RealRegister::Last8BitGPR;
          break;
+      case TR_DoubleWordReg:
+#if defined(TR_TARGET_X86)
```

Comment:
This #ifdef could be removed.  This is code in the x86 code generator.

Reply:
Isn't this code common to 32 and 64 bit?

Reply:
Yes, but `TR_TARGET_X86` applies to both.  If you want to make it specific to 32-bit, for example, you also need to use `defined (TR_TARGET_32BIT)` or `defined(TR_TARGET_64BIT)` for 64-bit.